### ISSUE TITLE
Constant client crashing

### ISSUE THREAD

**Issue by NinjaKittens14** [runelite/runelite#5372]
After a few minutes of playing OSRS, runelite just spontaneously closes. The timing of the crash is variable, but it happens in about 5 minutes. I have not been able to find a log file for runelite to obtain any crash error code, etc. Please help.

Computer Specs:
i7 6700k
Nvidia 980 ti
16GB Corsair Vengence
Samsung 850 evo 512GB SSD

**Comment by NathenSample** [comment]
Were you unable to find any logs in the logging folder; or do you mean you were unable to find the logging folder?

If it was the latter; then to copy the discord instructions - 
```To find logs, either open screenshot directory (by right-clicking "Camera" button) and navigate 1 directory up and then open logs folder, or navigate to %userprofile%\.runelite\logs on Windows or $HOME/.runelite/logs on Linux and macOS.```

**Comment by NinjaKittens14** [comment]
I apologize for not looking at the discord information. I did find the logs
with your help. Thank you! I have attached the logs from a crash today that
happened. I only played once today so there isn't much in the logs right
now.

**Comment by Nightfirecat** [comment]
Unfortunately we don't see the attachment (I don't think GitHub uploads those for us to see)--can you upload your logs to something like pastebin and send us that link so we can see them?

**Comment by NinjaKittens14** [comment]
I hope it works this way. Let me know.

[launcher.log](https://github.com/runelite/runelite/files/2362668/launcher.log)
[application.log](https://github.com/runelite/runelite/files/2362669/application.log)

**Comment by NinjaKittens14** [comment]
I tried to play again and it crashed again pretty quickly, meaning more are in the logs for you guys to look at to hopefully help. They are attached.

[launcher.log](https://github.com/runelite/runelite/files/2362714/launcher.log)
[application.log](https://github.com/runelite/runelite/files/2362715/application.log)

**Comment by Nightfirecat** [comment]
Looks like a JVM crash, since there are no error or crash entries in those logs.

**Comment by NinjaKittens14** [comment]
Alright, I'll look into Java VM crashing. Thank you guys very much!

**Comment by NathenSample** [comment]
Are there any hs_err files in the directory that your runelite jar sits in?

**Comment by NinjaKittens14** [comment]
No, there are no logs in the same directory as the jar file.

**Comment by NinjaKittens14** [comment]
I have determined that it was actually the latest Nvidia graphics driver that has been causing this issue. I rolled back the driver and it works perfectly now. Thank you guys again for the help!

**Comment by NathenSample** [comment]
Out of morbid curiosity could you include the resource that lead to that conclusion

**Comment by NinjaKittens14** [comment]
I think this issue was pretty situational to my graphics card. After the latest graphics card driver update, I noticed a few issues in other games as well, but kind of chalked it up to needing to reinstall windows (there has been issue growing from BSODs for me). After noticing Runelite crashes, I looked into why Java VM would crash and apparently there are very minimal circumstances were Java VM should ever crash. Stack overflow and other websites suggested that it would be a driver issue that would cause a Java VM crash, so I decided to roll back the driver. That roll back pretty much fixed all those issues from BSODs to Java VM crashes. I think that Nvidia stopped supporting 900 series cards at this point and doesn't really test for bugs on 900 series cards with their newer drivers. I hope this helps.
